src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationCommentsAfterMethodCall.java

```diff
@@ -0,0 +1,27 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+/* Config: default */
+
+public class InputCommentsIndentationCommentsAfterMethodCall {
+    public static void main() {
+        InputCommentsIndentationCommentsAfterMethodCall.flaMap(4,5)
+           // ok
+           .flaMap()
+           .flaMap();
+        // ok
+    }
+    public static void main1() {
+        InputCommentsIndentationCommentsAfterMethodCall.flaMap(4,5)
+                // ok
+                .flaMap()
+                .flaMap();
+               // violation
+    }
+    public static InputCommentsIndentationCommentsAfterMethodCall flaMap(int i, int h) {
+        return new InputCommentsIndentationCommentsAfterMethodCall();
+    }
+    public static InputCommentsIndentationCommentsAfterMethodCall flaMap() {
+        return new InputCommentsIndentationCommentsAfterMethodCall();
+    }
```

Comment:
Are these test cases required since there are no comments in both of these cases?

Reply:
These are part of the test cases...
You can look into this [example](https://github.com/checkstyle/checkstyle/issues/9105#issue-767009504) .I kind of tried to make similar example like this.

Reply:
Do they cover any branches? The reason is this check verifies the indentation of a comment, in absence of comment these lines are not even checked. The example you referred contains comments, so that is a valid test case. I am not able to understand how are these any test cases?

Reply:
No Akash they are not covering any branches. These are just method definition. If these are not defined its going to get compilation error.

Truly speaking methods are not actually serving any purpose in testing....just to avoid  compilation error.
Branches are covered only in the main method.

Reply:
Yup fine, my bad :)  A single typo make `flaMap()` -> `flatMap()`

Reply:
Done

Reply:
@AkMo3 I have just one doubt. This PR solves two issues .I have linked those two issues via PR description.
Should I do anything more to it?

Reply:
This would be better answered by maintainers. 
According to me, I think it is better to close #9945 as it is a duplicate of #9105 (Since 9105 was detected earlier), after doing a comment in 9945 that is a duplicate of 9105.